wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/servlet/handler/Responder.java

```diff
@@ -0,0 +1,88 @@
+package com.cognifide.aem.stubs.wiremock.servlet.handler;
+
+import static com.github.tomakehurst.wiremock.common.Exceptions.throwUnchecked;
+import static com.github.tomakehurst.wiremock.servlet.WireMockHttpServletRequestAdapter.ORIGINAL_REQUEST_KEY;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.tomakehurst.wiremock.http.HttpHeader;
+import com.github.tomakehurst.wiremock.http.HttpResponder;
+import com.github.tomakehurst.wiremock.http.Request;
+import com.github.tomakehurst.wiremock.http.Response;
+import com.github.tomakehurst.wiremock.verification.LoggedRequest;
+import com.google.common.io.ByteStreams;
+
+class Responder implements HttpResponder {
```

Comment:
this copied or impl on your own?

Reply:
copied from previous servlet in-class impl. decided to have it separately.